mlir/lib/Dialect/Vector/Transforms/LowerVectorBroadcast.cpp

```diff
@@ -28,7 +28,10 @@ using namespace mlir;
 using namespace mlir::vector;
 
 namespace {
-/// Progressive lowering of BroadcastOp.
+
+/// Convert a vector.broadcast without a scalar operand to a lower rank
```

Comment:
[nit] This feels more natural to me 🤷🏻 
```suggestion
/// Convert a vector.broadcast with a vector operand to a lower rank
```